### PR TITLE
[linux] Linux options

### PR DESCRIPTION
**Pull request by glasseyes**
adds `platform "linux desktop hardware native"`
`baselayout "kbdus.dll"`
`baselayoutAlt "en-US"`

---
This change is [Reviewable](https://reviewable.io/reviews/keymanapp/keyman/1446)

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/137589:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/137599:id/kvk/index.html)

File: linux/ibus-keyman/src/engine.c [hunk 1]
```diff
@@ -324,7 +324,47 @@ ibus_keyman_engine_constructor (GType                   type,
...
+            lang = g_strdup(splitlang[0]);
+        }
+        g_strfreev(splitlang);
+    }
+    else {
+        lang = strdup("en-US");
+    }
+    g_message("lang is %s", lang);
+    #endif
+    cp = g_utf8_to_utf16 ("en-US", -1, NULL, NULL, NULL);
```
Comment (mcdurdin) [inline linux/ibus-keyman/src/engine.c]: `baseLayout` should be `kbdus.dll` and `baseLayoutAlt` should be `en-us`, for this release :)

Comment (glasseyes) [reply linux/ibus-keyman/src/engine.c]: done :)

**Pull request merged**